config.json

```diff
@@ -14,6 +14,16 @@
         "strings"
       ]
     },
+    {
+      "slug": "two-fer",
```

Comment:
Maybe it is a good idea to have this exercise also unlock one or two other exercises?

Reply:
Good point. thanks